config/local/market.json

```diff
@@ -702,2122 +702,2126 @@
     "ADA/USD": {
       "providers": [
         {
-          "name": "okx",
```

Comment:
Thank god we use constants everywhere instead of hardcoding names, all of this has to be updated since the provider names changed.